Tests/DeepLearningTests/TrivialModelTests.swift

```diff
@@ -34,7 +34,7 @@ final class TrivialModelTests: XCTestCase {
                 )
             }
             @differentiable
-            func applied(to input: Tensor<Float>) -> Tensor<Float> {
+            call func(_ input: Tensor<Float>) -> Tensor<Float> {
                 let h1 = l1.applied(to: input)
```

Comment:
What about call-sites?
```suggestion
                let h1 = l1(input)
```

Reply:
Sure. Still building a toolchain.